src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Api/Altinn.App.Api.csproj

```diff
@@ -3,16 +3,16 @@
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
     <LangVersion>latest</LangVersion>
-    <Version>4.0.0.1-alpha</Version>
+    <Version>4.0.0</Version>
```

Comment:
I think this needs to be at least 4.0.0.1 now that we have had an alpha version.

Reply:
This is what we've practiced for other packages that we iterate over a level lower than patch version when we have changes during testing.. 
I don't think we should push final versions with anyting other that major.minor.patch

Reply:
Then I suggest more specifically 4.0.1. We have already published a package with a version higher then 4.0.0.